### PR TITLE
Create route for nginx

### PR DESCRIPTION
**Pull request by bigkevmcd**
/kind enhancement
**What does this PR do / why we need it**:
A new Route is created during bootstrapping to make the default generated Nginx container available for testing.

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: pkg/pipelines/eventlisteners/routes.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package eventlisteners
```
Comment (wtam2018) [inline pkg/pipelines/eventlisteners/routes.go:1]: This module does not have much to do with eventlisteners.

**Review by wtam2018 — COMMENTED** [review]
Please update documentation.

File: pkg/pipelines/eventlisteners/routes.go (hunk 1 above)
Comment (bigkevmcd) [reply pkg/pipelines/eventlisteners/routes.go:1]: This package is for creating an EventListener, and this creates the route to point to it.

The EventListener creates its own service, so there's no service to create a route for.

**Comment by keithchong** [comment]
Test considerations:

When you bootstrap a configuration. a new file 300-route.yaml is created in (for the taxi example):

environments/{prefix-}dev/apps/app-taxi/services/taxi/base/config

When you apply the resources and check ArgoCD, a new route resource is created for the service.  Note that the service should be synchronized.

**Pull request merged**